omero/autogen_docs

```diff
@@ -32,8 +32,8 @@ echo "Generating DB script example"
 
 echo "Generating Web configuration templates"
 # Nginx / WSGI
-$WORKSPACE/OMERO.server/bin/omero web config nginx | sed "s|$WORKSPACE|/home/omero|g" > omero/sysadmins/unix/install-web/nginx-omero.conf
-$WORKSPACE/OMERO.server/bin/omero web config nginx-location | sed "s|$WORKSPACE|/home/omero|g" | grep -v '^#' > omero/sysadmins/unix/install-web/nginx-location.conf
+$WORKSPACE/OMERO.server/bin/omero web config nginx | sed "s|$WORKSPACE/OMERO.server|/home/omero/OMERO.py|g" > omero/sysadmins/unix/install-web/nginx-omero.conf
+$WORKSPACE/OMERO.server/bin/omero web config nginx-location | sed "s|$WORKSPACE/OMERO.server|/home/omero|g" | grep -v '^#' > omero/sysadmins/unix/install-web/nginx-location.conf
```

Comment:
I think you also need `/home/omero` -> `/home/omero/OMERO.py`, though admittedly it's difficult to check if this isn't actually used.

Reply:
sorry you are correct.Doing too many things at once